sysvabi64/sysvabi64.rst

```diff
@@ -1592,6 +1592,163 @@ IFUNC requirements for dynamic linkers
+    +-------------------+----------+
+    | FEAT_RCPC2        | 1U << 24 |
+    +-------------------+----------+
+    | FEAT_FRINTTS      | 1U << 25 |
+    +-------------------+----------+
+    | FEAT_I8MM         | 1U << 27 |
+    +-------------------+----------+
+    | FEAT_BF16         | 1U << 28 |
+    +-------------------+----------+
+    | FEAT_SVE          | 1U << 31 |
+    +-------------------+----------+
+    | FEAT_SVE_F32MM    | 1U << 35 |
+    +-------------------+----------+
+    | FEAT_SVE_F64MM    | 1U << 36 |
+    +-------------------+----------+
+    | FEAT_SVE2         | 1U << 37 |
+    +-------------------+----------+
+    | FEAT_SVE_PMULL128 | 1U << 39 |
+    +-------------------+----------+
+    | FEAT_SVE_BITPERM  | 1U << 40 |
+    +-------------------+----------+
+    | FEAT_SVE_SHA3     | 1U << 41 |
+    +-------------------+----------+
+    | FEAT_SVE_SM4      | 1U << 42 |
+    +-------------------+----------+
+    | FEAT_SME          | 1U << 43 |
+    +-------------------+----------+
+    | FEAT_MEMTAG2      | 1U << 45 |
+    +-------------------+----------+
+    | FEAT_SB           | 1U << 47 |
+    +-------------------+----------+
+    | FEAT_SSBS2        | 1U << 50 |
+    +-------------------+----------+
+    | FEAT_BTI          | 1U << 51 |
+    +-------------------+----------+
+    | FEAT_WFXT         | 1U << 55 |
+    +-------------------+----------+
+    | FEAT_SME_F64      | 1U << 56 |
+    +-------------------+----------+
+    | FEAT_SME_I64      | 1U << 57 |
+    +-------------------+----------+
+    | FEAT_SME2         | 1U << 58 |
+    +-------------------+----------+
+    | FEAT_RCPC3        | 1U << 59 |
+    +-------------------+----------+
+    | FEAT_MOPS         | 1U << 60 |
+    +-------------------+----------+
+
+A special value ``FEAT_INIT = (1U << 63);`` is used to signify
+initialization completion.
+
+Implementing FMV using ``__aarch64_cpu_features`` is not required.
+Accessing ``__aarch64_cpu_features`` from outside a FMV resolver
+function is not well defined.
+
+Initialization of the ``__aarch64_cpu_features`` is performed by
+the runtime library. Each FMV resolver function will call the
+``__init_cpu_features_resolver`` function at load time. This runtime
+function must ensure that ``__aarch64_cpu_features`` is populated.
+
+Systems without support for IFUNC may call the
+``__init_cpu_features_resolver`` function from an
+``__attribute__((constructor))`` function provided by the runtime.
+
+Systems with support for IFUNC should place the global variable
+``__aarch64_cpu_features`` in Relocation Read Only (RELRO) segment
+to prevent it from being modified after the FMV resolvers have run.
+
+The ``__init_cpu_features_resolver`` function has the following
+prototype:
+
+.. code-block:: c
+
+   void __init_cpu_features_resolver (uint64_t, const uint64_t *);
+
+The above interface expects the same parameters as a GNU Indirect
+Function resolver. Other platforms may use a different interface
```

Comment:
Can you add a reference to the section where that is defined with `GNU C Library IFUNC interface`_ ?